text/metrics/0238-pipeline-monitoring.md

```diff
@@ -0,0 +1,426 @@
+removed, loss-rate can likewise only be calculated in aggregate.
+
+#### Pipeline component name
+
+Components are uniquely identified using a descriptive `name`
+attribute which encompasses at least a short name describing the type
+of component being used (e.g., `batch` for the SDK BatchSpanProcessor
+or the Collector batch proessor).
+
+When there is more than one component of a given type active in a
+pipeline having the same `domain` and `signal` attributes, the `name`
+should include additional information to disambiguate the multiple
+instances using the syntax `<type>/<instance>`.  For example, if there
+were two `batch` processors in a collection pipeline (e.g., one for
+error spans and one for non-error spans) they might use the names
+`batch/error` and `batch/noerror`.
+
+#### Pipeline monitoring diagram
+
+The relationship between items received, dropped, and exported is
+shown in the following diagram.
+
+![pipeline monitoring metrics](../images/otel-pipeline-monitoring.png)
+
+### Proposed metrics semantic conventions
+
+The proposed metric names match the following pattern:
+
+| Metric Name               | Meaning                                                            |
+|---------------------------|--------------------------------------------------------------------|
+| `otel.{station}.received` | Inclusive count of items entering the pipeline at a station.       |
+| `otel.{station}.dropped`  | Non-inclusive count of items dropped by a processor in a pipeline. |
+| `otel.{station}.exported` | Inclusive count of items exiting the pipeline at a station.        |
+
+The behavior specified for SDKs and Collectors at each level of detail
+is different, because SDKs do not receive items from a pipeline and
+because they outnumber the other components.
+
+These attributes can be applied to any of the pipeline monitoring
+metrics specified here.
+
+| Attributes     | Meaning                                     | Level of detail (Optional) | Examples                                                   |
+|----------------|---------------------------------------------|----------------------------|------------------------------------------------------------|
+| `otel.signal`  | Name of the telemetry signal                | Basic (Opt-out)            | `traces`, `logs`, `metrics`                                |
+| `otel.name`    | Type, name, or "type/name" of the component | Normal (Opt-out)           | `probabilitysampler`, `batch`, `otlp/grpc`                |
+| `otel.success` | Boolean: item considered success?           | Normal (Opt-out)           | `true`, `false`                                            |
+| `otel.reason`  | Explaination of success/failures.           | Detailed (Opt-in)          | `ok`, `timeout`, `permission_denied`, `resource_exhausted` |
+| `otel.scope`   | Name of instrumentation.                    | Detailed (Opt-in)          | `opentelemetry.io/library`                                 |
```

Comment:
Is this intended to be different than the scope name? It seems like its the same. If so, then we can't opt-in or out of it. Maybe the prometheus exporter has options for that, but the scope isn't an optional part of the data model.